### PR TITLE
Add internal label for Kyma resource

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

Changes proposed in this pull request:

- Set `operator.kyma-project.io/internal` to true during provisioning and upgrading Kyma for internal users

**Related issue(s)**

See also #3005

### CODE REVIEW

File: components/kyma-environment-broker/internal/process/provisioning/apply_kyma_step_test.go [hunk 1]
```diff
@@ -81,13 +105,13 @@ func TestCreatingKymaResource_UseNamespaceFromTimeOfCreationNotTemplate(t *testi
...
 	assert.Equal(t, 1, len(aList.Items))
-	assertLabelsExists(t, aList.Items[0])
+	assertLabelsExistsForInternalKymaResource(t, aList.Items[0])
 
 	svc.Run(operation, logrus.New())
 	assert.Equal(t, "namespace-in-time-of-creation", operation.KymaResourceNamespace)
 }
 
-func TestUpdatingKymaResourceIfExists(t *testing.T) {
+func TestUpdatinInternalKymaResourceIfExists(t *testing.T) {
```
Comment (jaroslaw-pieszka) [inline components/kyma-environment-broker/internal/process/provisioning/apply_kyma_step_test.go:114]: Is there the negative test? Testing if Kyma resource is not marked as internal.

Comment (MarekMichali) [reply components/kyma-environment-broker/internal/process/provisioning/apply_kyma_step_test.go:114]: No. I can add.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**